README.md

```diff
@@ -1,31 +1,120 @@
-2. [reference-architecture](https://github.com/Azure-Samples/ASA-Samples-Web-Application/blob/reference-architecture/README.md). This branch give the reference architecture (best practice) of running spring web application in Azure Spring Apps.
+A blueprint for getting a React web app with a Java API and a PostgreSQL - Flexible Server on Azure. The blueprint includes sample application code (a ToDo web app) which can be removed and replaced with your own application code. Add your own source code and leverage the Infrastructure as Code assets (written in Bicep) to get up and running quickly. This architecture is for running containerized apps or microservices on a serverless platform.
 
-## Prerequisites
+Let's jump in and get this up and running in Azure. When you are finished, you will have a fully functional web app deployed to the cloud. In later steps, you'll see how to setup a pipeline and run the application.
 
-- Java 17 or later
+!["Screenshot of deployed ToDo app"](assets/web.png)
 
-## Run the app in localhost
+<sup>Screenshot of the deployed ToDo app</sup>
 
-1. Build sample project.
+### Prerequisites
 
-    ```shell
-    ./mvnw clean package -DskipTests
-    ```
+The following prerequisites are required to use this application. Please ensure that you have them all installed locally.
 
-2. Run sample project.
+- [Azure Developer CLI](https://aka.ms/azd-install)
+- [Java 17 or later](https://learn.microsoft.com/en-us/java/openjdk/install) - for API backend
+- [Node.js with npm (16.13.1+)](https://nodejs.org/) - for the Web frontend
+- [Docker](https://docs.docker.com/get-docker/)
 
-    ```shell
-    ./mvnw spring-boot:run -f web/pom.xml
-    ```
+### Quickstart
 
-3. Access `http://localhost:8080` by browser, you will see a page like this:
+To learn how to get started with any template, follow the steps in [this quickstart](https://learn.microsoft.com/azure/developer/azure-developer-cli/get-started?tabs=localinstall&pivots=programming-language-java) with this template(`Azure-Samples/ASA-Samples-Web-Application`).
 
-> ![web.png](./assets/web.png)
+This quickstart will show you how to authenticate on Azure, initialize using a template, provision infrastructure and deploy code on Azure via the following commands:
 
-## Run the app in Azure Spring Apps.
+```bash
+# Log in to azd. Only required once per-install.
+azd auth login
 
-Please refer to [Quickstart: Launch your first web app](.) (todo: Update the link when the article is ready.) to get more information about running the app in Azure Spring Apps.
+# Enable Azure Spring Apps feature for AZD
+azd config set alpha.springapp on
+
+# First-time project setup. Initialize a project in the current directory, using this template. 
+azd init --template Azure-Samples/ASA-Samples-Web-Application
+
+# Provision and deploy to Azure
+azd up
+```
+
+> NOTE: This template may only be used with the following Azure locations:
```

Comment:
shoule we remove it? keep same pattern with event-driven README

Reply:
we can remove it, seems like I referenced different templates for the two samples.